File: 프로그래머스/1/132267. 콜라 문제/콜라 문제.java
class Solution {
   public int solution(int a, int b, int c) {
        int answer = 0 ;
        while (c >= a) {
             int bottle = (c/a)*b;
            int div = c%a ;
            answer += bottle;
            c = bottle+div;

        }
        return answer ;
    }


}

File: 프로그래머스/1/132267. 콜라 문제/콜라 문제.js
function solution(a, b, n) {
    var result = 0 
    var answer = 0;
    while(n>=a){
        answer = Math.floor(n / a) * b ;
        n = n - (Math.floor(n/a) * a) + answer   ; 
        result += answer ; 
        
    }
    return result;
}
